lib/src/core/message/guild_emoji.dart

```diff
@@ -14,41 +15,87 @@ abstract class IBaseGuildEmoji implements SnowflakeEntity, IEmoji {
   /// Returns cdn url to emoji
   String get cdnUrl;
 
+  /// The name of the emoji.
+  String get name;
+
+  /// Whether this emoji is animated.
+  bool get animated;
+
   /// Creates partial emoji from given String or Snowflake.
-  factory IBaseGuildEmoji.fromId(Snowflake id) => GuildEmojiPartial(id);
```

Comment:
To avoid breaking changes, could you keep the `fromId` constructor and have it return a partial emoji that can be resolved with the client's emoji cache?

Reply:
Yeah, sorry, I didn't think about it at all

Reply:
Fixed!